mypy/nodes.py

```diff
@@ -1242,8 +1241,6 @@ def accept(self, visitor: ExpressionVisitor[T]) -> T:
 class ComplexExpr(Expression):
     """Complex literal"""
 
-    value = 0.0j
-
```

Comment:
So complex numbers are not yet supported in mypyc? I would actually implement them as an exercise in mypyc. (Next exercise is implementing full SSA :-))

Reply:
They should work, but complex literals aren't implemented yet. Sounds good!